Main/Source/char.cpp

```diff
@@ -6208,7 +6208,7 @@ void character::DrawPanel(truth AnimationDraw) const
 
   ivantime Time;
   game::GetTime(Time);
-  FONT->Printf(DOUBLE_BUFFER, v2(PanelPosX, PanelPosY++ * 10), WHITE, "Day %d", Time.Day);
+  FONT->Printf(DOUBLE_BUFFER, v2(PanelPosX, PanelPosY++ * 10), WHITE, "Day  %d", Time.Day);
```

Comment:
Why adding extra space?

Reply:
To align the number with the numbers below it. It might be just my preference, but I think it looks cleaner when the text and numbers are aligned in two distinct columns.